src/utils/hooks/UserWindow.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react'
+
+export default function UserWindow() {
+    const [screenSize, getDimension] = useState({
+        windowWidth: window.innerWidth,
+        windowHeight: window.innerHeight
+    });
+    const setDimension = () => {
+        getDimension({
+            windowWidth: window.innerWidth,
+            windowHeight: window.innerHeight
+        })
+    }
+
+    useEffect(() => {
+        window.addEventListener('resize', setDimension);
+
+        return (() => {
+            window.removeEventListener('resize', setDimension);
+        })
+    }, [screenSize])
+    return (screenSize)
```

Comment:
Excelente custom hook!, in some projects, a map of units is also used in order to have standard breakpoints, so the media queries can be coded easily, good job!